Makefile.include

```diff
@@ -139,8 +139,8 @@ api-test:             ## Run API tests on dev env. Use `PMM_KUBECONFIG=/path/to/
 	go test -count=1 -race -p 1 -v ./api-tests/... -pmm.server-insecure-tls
 
 check:                ## Run required checkers and linters
-	LOG_LEVEL=error bin/golangci-lint run
-	bin/go-sumtype ./...
+	LOG_LEVEL=error bin/golangci-lint run ; \
```

Comment:
without this change task is stopped on first command failure, i.g. if `golangci-lint` return error go-sumtype and go-consistent are not run.

before change:
<img width="1666" alt="image" src="https://user-images.githubusercontent.com/91831719/224575461-043b8efd-4c68-41c4-a278-ff873ff57523.png">

after change:
<img width="1656" alt="image" src="https://user-images.githubusercontent.com/91831719/224575465-2d9833a1-a16f-4fc5-8daf-58624d6acb50.png">